src/main/java/org/phoebus/channelfinder/ElasticConfig.java

```diff
@@ -98,11 +114,32 @@ public int getES_QUERY_SIZE() {
             .addMixIn(Property.class, Property.OnlyProperty.class);
 
     private static ElasticsearchClient createClient(ElasticsearchClient currentClient, ObjectMapper objectMapper,
-                                                    String host, int port, String createIndices, ElasticConfig config) {
+                                                    List<String> hostUrls, String createIndices, ElasticConfig config) {
         ElasticsearchClient client;
         if (currentClient == null) {
             // Create the low-level client
-            RestClient httpClient = RestClient.builder(new HttpHost(host, port)).build();
+            HttpHost[] httpHosts = new HttpHost[hostUrls.size()];
+            for (int i = 0; i < httpHosts.length; ++i) {
```

Comment:
You can use List.toArray here, or as mentioned us an array SpeL expression for elasticsearch.host_urls